packages/mdc-fab/mdc-fab.scss

```diff
@@ -12,98 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@import "@material/animation/functions";
-@import "@material/animation/variables";
-@import "@material/elevation/mixins";
-@import "@material/ripple/mixins";
-@import "@material/theme/mixins";
+@import "mixins";
```

Comment:
Seems like we are typically explicit and reference `./mixins` for imports in the same folder.